src/table/utils/__tests__/color-utils.spec.ts

```diff
@@ -197,8 +196,8 @@ describe('color-utils', () => {
       const result = removeOpacity(color);
       expect(result).toBe('rgb(0,0,0)');
     });
-    it('should return undefined when nothing is provided', () => {
-      const result = removeOpacity();
+    it('should return undefined when color is undefined', () => {
```

Comment:
'should return undefined when nothing is provided' is also a valid test case I think

Reply:
No, when this is typed it always wants an argument. so `removeOpacity()` is invalid. You can still do `removeOpacity(someVariable)` where the variable is undefined

Reply:
aha, color: string | undefined. I see.